CMakeLists.txt

```diff
@@ -188,6 +188,9 @@ if(MSVC)
   add_link_options("$<$<NOT:$<CONFIG:Debug>>:/OPT:ICF>")
 endif()
 
+
+find_package(ZXing REQUIRED)
```

Comment:
Can you move that into src/core/CMakeLists.txt?

Reply:
Done.